airflow/api_fastapi/core_api/routes/public/assets.py

```diff
@@ -69,3 +69,20 @@ def get_assets(
         assets=[AssetResponse.model_validate(asset, from_attributes=True) for asset in assets],
         total_entries=total_entries,
     )
+
+
+@assets_router.get(
+    "/{uri:path}",
+    responses=create_openapi_http_exception_doc([401, 403, 404]),
+)
+async def get_asset(
+    uri: str,
+    session: Annotated[Session, Depends(get_session)],
+) -> AssetResponse:
+    """Get an asset."""
+    asset = session.scalar(select(AssetModel).where(AssetModel.uri == uri))
```

Comment:
I think you removed the loading options. This will make the the serialization step slower by emitting two additionnal lazy db query.

Reply:
This makes me realize that you also removed that part on the `get_assets`.

Can you please add it back to both of them and make sure to not remove anything if you'r not sure of the impact. (here that could drastically slow down the endpoint)

Reply:
Sure, looks like I missed it. Added and ran unit tests, it works as expected.